
This seems to be required to bring back audio on Opi Zero2 as mainline rtc dt
gives no sound on Opi Zero2...


diff --speed-large-files --no-dereference --minimal -Naur linux-6.2.2/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi linux-6.2.2/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
--- linux-6.2.2/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi	2023-03-07 12:16:16.848060486 +0100
+++ linux-6.2.2/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi	2023-03-07 12:00:30.001428364 +0100
@@ -1133,13 +1132,12 @@
 		};
 
 		rtc: rtc@7000000 {
-			compatible = "allwinner,sun50i-h616-rtc";
+			compatible = "allwinner,sun50i-h616-rtc",
+				     "allwinner,sun50i-h6-rtc";
 			reg = <0x07000000 0x400>;
-			interrupts = <GIC_SPI 104 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&r_ccu CLK_R_APB1_RTC>, <&osc24M>,
-				 <&ccu CLK_PLL_SYSTEM_32K>;
-			clock-names = "bus", "hosc",
-				      "pll-32k";
+			interrupts = <GIC_SPI 101 IRQ_TYPE_LEVEL_HIGH>,
+				     <GIC_SPI 102 IRQ_TYPE_LEVEL_HIGH>;
+			clock-output-names = "osc32k", "osc32k-out", "iosc";
 			#clock-cells = <1>;
 		};
 
